lib/media/Photo.js

```diff
@@ -241,15 +255,30 @@ export default class Photo extends Component {
     return (
       <View style={[styles.container, sizeStyle]}>
         {error ? this._renderErrorIcon() : this._renderProgressIndicator()}
-        <TransformableImage
-          {...this.props}
-          style={[styles.image, sizeStyle]}
-          source={source}
-          onProgress={this._onProgress}
-          onError={this._onError}
-          onLoad={this._onLoad}
-          resizeMode={resizeMode}
-        />
+        {
+          transformable ? (
+            <TransformableImage
```

Comment:
used this lib "react-native-transformable-image", but this has a few of conflict on pan gesture when scrolling, because it's implemented by scroll view too!